src/runtime_src/core/common/utils.cpp

```diff
@@ -180,6 +180,26 @@ unit_convert(size_t size)
   return ret+" "+unit[i-1];
 }
 
+std::string
```

Comment:
Boost format already does this:
`boost::format("0x%x" % size);  `
Is there a reason why it can't be used.

Reply:
done